SYCL/ESIMD/api/functional/value_conv.hpp

```diff
@@ -0,0 +1,91 @@
+//===-- value_conv.hpp - This file provides common functions generate values for
+//      testing. ----------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functions that let obtain converted reference data for
+/// test according to current underlying types.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "value.hpp"
+
+namespace esimd_test::api::functional {
+
+// Utility class to retrieve specific values for tests depending on the source
+// and destination data types. May be used to retrieve converted reference data.
+// All provided methods are save to use and protected from UB when call
```

Comment:
```suggestion
// All provided methods are safe to use and protected from UB when call
```
Not clear what the last sentence actually means, please rephrase,

Reply:
I've applied this comment in this commit: 00693ecea35d0a2935c72bb247a7a9835f965a42